zk-token-sdk/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(clippy::integer_arithmetic)]
+#![allow(clippy::op_ref)]
```

Comment:
Any way this allow can be remove as well?  Generally clippy allows are a bad code smell, especially when applied to the entire module

Reply:
Ah, I see.  A brief comment about why this is allowed in the code would be useful then!

Reply:
Great, will do!